CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ Currently the versioning policy of this project follows [Semantic Versioning v2.
 * Handle annotation on `package-info.class` properly ([#592](https://github.com/spotbugs/spotbugs/issues/592))
 * Update asm to 6.1.1 to support Java 10
 * Update Apache BCEL to 6.2 to support Java 9 package & module reference
+* RANGE_ARRAY_LENGHT and RANGE_ARRAY_OFFSET false negative ([#595](https://github.com/spotbugs/spotbugs/issues/595))
```

Comment:
Please merge latest `release-3.1`, solve conflict, and move this line to under `## Unreleased - 2018-??-??`.